statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -1196,23 +1224,22 @@ def _update_assoc(self, params):
     def _derivative_exog(self, params, exog=None, transform='dydx',
             dummy_idx=None, count_idx=None):
         """
-        For computing marginal effects returns dF(XB) / dX where F(.) is
-        the predicted probabilities
+        For computing marginal effects, returns dF(XB) / dX where F(.)
+        is the fitted mean.
 
         transform can be 'dydx', 'dyex', 'eydx', or 'eyex'.
 
         Not all of these make sense in the presence of discrete regressors,
         but checks are done in the results in get_margeff.
         """
-        #note, this form should be appropriate for
-        ## group 1 probit, logit, logistic, cloglog, heckprob, xtprobit
+        # This form should be appropriate for group 1 probit, logit,
+        # logistic, cloglog, heckprob, xtprobit.
         if exog is None:
             exog = self.exog
-        margeff = self.mean_deriv_exog(exog, params)
-#        lpr = np.dot(exog, params)
-#        margeff = (self.mean_deriv(exog, lpr) / exog) * params
-#        margeff = np.dot(self.pdf(np.dot(exog, params))[:, None],
-#                                                          params[None,:])
+            offset_exposure = self._offset_exposure
+
+        offset_exposure = None
```

Comment:
My guess is that this is consistent with discrete models right now, where offset and exposure are ignored, AFAICS.

We need to check and test the behavior of margins with respect to offset and exposure. I lost my overview over the Margin code again.